src/main/java/core/basesyntax/dao/ReadDao.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dao;
+
+import java.util.List;
+
+public interface ReadDao {
```

Comment:
let's not name these classes related to the file read and write operations as Dao. I'd recommend move them to the service package. Also, let's add to all your service names a 'Service' suffix